testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/system/system_email.t.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests system email server viewing, updating, discarding.
+ */
+StartTest(function(t) {
+  var originalValues;
+
+  t.describe('Given an admin administering Email Server', function(t) {
+    t.it('Should be able to view the configuration', function(t) {
+      t.chain(
+          t.openPageAsAdmin('admin/system/emailserver'),
+          {wait: 'CQVisible', args: '>>nx-coreui-system-smtp-settings'},
+          // capture the values to use later to verify, discard, and reset form
+          function(next) {
+            var settings = t.cq1('nx-coreui-system-smtp-settings'),
+                form = settings.down('form').getForm();
+            originalValues = form.getFieldValues();
+            next();
+          }
+      );
+    });
+    t.it('Should be able to discard settings on the form', function(t) {
+      t.chain(
+          // type in some test strings to empty fields
+          t.selectAndType('_test', '>>nx-coreui-system-smtp-settings field[name=subjectPrefix]'),
+          t.selectAndType('_testUser', '>>nx-coreui-system-smtp-settings field[name=username]'),
+          t.selectAndType('_testPassword', '>>nx-coreui-system-smtp-settings field[name=password]'),
+          {click: '>>nx-coreui-system-smtp-settings button[text=Discard]', desc: 'Discard all changes'},
```

Comment:
looks like SettingsForm doesn't have an action for this button, but it does for save.   Should be updated:

```
diff --git a/plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/SettingsForm.js b/plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/SettingsForm.js
index c0216fc..5d2d5be 100644
--- a/plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/SettingsForm.js
+++ b/plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/SettingsForm.js
@@ -93,7 +93,9 @@ Ext.define('NX.view.SettingsForm', {
 
   buttons: [
     { text: NX.I18n.get('SettingsForm_Save_Button'), formBind: true, action: 'save', ui: 'nx-primary', bindToEnter: false },
-    { text: NX.I18n.get('SettingsForm_Discard_Button'),
+    {
+      text: NX.I18n.get('SettingsForm_Discard_Button'),
+      action: 'discard',
       handler: function () {
         var form = this.up('form'),
             record = form.getRecord();
```

... and the query here changed to `button[action=discard]`